neqo-transport/src/recv_stream.rs

```diff
@@ -517,12 +529,27 @@ impl RecvStream {
         tokens: &mut Vec<RecoveryToken>,
         stats: &mut FrameStats,
     ) -> Res<()> {
-        // Send MAX_STREAM_DATA at normal priority always.
-        if let RecvStreamState::Recv { fc, .. } = &mut self.state {
-            fc.write_frames(builder, tokens, stats)
-        } else {
-            Ok(())
+        match &mut self.state {
+            // Maybe send MAX_STREAM_DATA
+            RecvStreamState::Recv { fc, .. } => fc.write_frames(builder, tokens, stats)?,
+            // Maybe send STOP_SENDING
+            RecvStreamState::ResetSending { pending, err } => {
+                if *pending
```

Comment:
"pending" might not be the right word here.  It could mean that you sending the frame is pending, or it could mean that the reset is pending.  Here, you have it meaning that the frame is pending, but as it is attached to something called "ResetSending", it would be easy to misinterpret.  I would go with frame_needed or something like that.